conf/map/help.txt

```diff
@@ -305,3 +305,5 @@ autoloottype: "Manage a list of autolooting item types.\n"
 " To remove an item type, use \"@autoloottype -<type name>\".\n"
 " Type List: healing, usable, etc, weapon, armor, card, petegg, petarmor, ammo.\n"
 " \"@autoloottype reset\" will clear your autoloottype list."
+feelreset: "Reset 'Feeling' maps."
```

Comment:
Here it says feelreset, but this command is not added in source

Reply:
This atcommand already exists in source.